package.json

```diff
@@ -147,6 +147,7 @@
     "yup": "^0.28.4"
   },
   "dependencies": {
+    "@types/pino": "^6.0.1",
```

Comment:
we intentionally omitted these types and provide instructions how to include them in the typescript documentation (see example 5 here: https://www.fastify.io/docs/latest/TypeScript/#fastify).

Reply:
I totally missed this. Definitely -1. Those types are not really maintained.

Reply:
Thanks for the info. The TypeScript docs are incredibly well written and detailed, but my suspicion is that this might come up again as users might expect some level of completion because Fastify ships with its own types. Anyway I removed the pino types and added a comment in code referencing the example.